traffic_portal/app/src/common/modules/form/cacheGroup/_form.cacheGroup.scss

```diff
@@ -0,0 +1,48 @@
+.allFallbacks{
+  margin-top: 10px;
+}
+
+.geoFallback {
+  padding-top: 8px;
+  padding-bottom: 20px;
+}
+
+.fallback {
+  border: 1px solid;
+  border-color: #ccc;
+  width: 100%;
+  height: 28px;
+  padding: 5px 8px;
+  font-size: 12px;
+  line-height: 1.428571429;
+  color: #555555;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+}
+
+.fallback-grip {
+  float:left;
+  padding-top: 4px;
+  padding-right: 10px;
+}
+
+.fallback-list {
+  display:inline;
+  float:left;
+  width:90%;
+  align-items:center;
+}
+
+.fallback-delete {
+  display:inline-block;
+  float:right;
+  width:5%;
+}
+
+.fallback:hover {
```

Comment:
You know, since this is a SCSS file you _could_ use SCSS to put this inside of the `.fallback` rule. It'd look something like:

```scss
.fallback {
    background-color: #fff;

    &:hover {
        background: rgba(38,185,154,0.07);
    }
}
```

Reply:
Just a suggestion. I think it's a real shame that we spent all that time and effort fixing the sass compiler when 90% of our styling doesn't even make use of it.

Reply:
done